<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>ModifyAccountWindow</class>
	<widget class="QWidget" name="ModifyAccountWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1024</width>
				<height>768</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Modify Account - CSE</string>
		</property>
		<property name="styleSheet">
			<string notr="true">
				/* General Background */
				QWidget#ModifyAccountWindow {
				background-color: #f0f4f8;
				}

				/* Header and Footer */
				QFrame#headerFrame, QFrame#footerFrame {
				background-color: #0c3b6d;
				border: none;
				}

				/* Header Text */
				QLabel#logoLabel {
				color: white;
				font-size: 20px;
				font-weight: bold;
				}

				/* Footer Text */
				QLabel#footerLinks {
				color: white;
				}

				/* Page Title */
				QLabel#pageTitle {
				color: #333;
				font-size: 24px;
				font-weight: 500;
				margin-bottom: 10px;
				}

				/* Card Container */
				QFrame#modifyCard {
				background-color: white;
				border: 1px solid #dcdcdc;
				border-top: 4px solid #0c3b6d;
				border-radius: 4px;
				}

				/* Inputs */
				QLineEdit {
				border: 1px solid #cccccc;
				border-radius: 4px;
				padding: 8px;
				font-size: 14px;
				color: #333;
				background-color: white;
				}
				QLineEdit:focus {
				border: 1px solid #0c3b6d;
				}

				/* Disabled Input (Account Number) */
				QLineEdit:disabled, QLineEdit[readOnly="true"] {
				background-color: #e9ecef;
				color: #6c757d;
				border: 1px solid #ced4da;
				}

				/* Labels */
				QLabel {
				color: #555;
				font-size: 13px;
				}

				/* Primary Button (Save Changes) */
				QPushButton#saveBtn {
				background-color: #0c3b6d;
				color: white;
				border-radius: 4px;
				padding: 10px;
				font-weight: bold;
				font-size: 14px;
				}
				QPushButton#saveBtn:hover {
				background-color: #0a2e55;
				}

				/* Small Update Buttons */
				QPushButton.updateBtn {
				background-color: #0c3b6d;
				color: white;
				border-radius: 4px;
				padding: 5px 15px;
				font-size: 12px;
				font-weight: bold;
				}
				QPushButton.updateBtn:hover {
				background-color: #0a2e55;
				}
			</string>
		</property>
		<layout class="QVBoxLayout" name="verticalLayout">
			<property name="spacing">
				<number>0</number>
			</property>
			<property name="leftMargin">
				<number>0</number>
			</property>
			<property name="topMargin">
				<number>0</number>
			</property>
			<property name="rightMargin">
				<number>0</number>
			</property>
			<property name="bottomMargin">
				<number>0</number>
			</property>

			<item>
				<widget class="QFrame" name="headerFrame">
					<property name="minimumSize">
						<size>
							<width>0</width>
							<height>70</height>
						</size>
					</property>
					<layout class="QHBoxLayout" name="horizontalLayout_2">
						<property name="leftMargin">
							<number>40</number>
						</property>
						<item>
							<widget class="QLabel" name="logoLabel">
								<property name="text">
									<string>A  Apex Trust Bank</string>
								</property>
							</widget>
						</item>
						<item>
							<spacer name="horizontalSpacer">
								<property name="orientation">
									<enum>Qt::Horizontal</enum>
								</property>
								<property name="sizeHint" stdset="0">
									<size>
										<width>40</width>
										<height>20</height>
									</size>
								</property>
							</spacer>
						</item>
					</layout>
				</widget>
			</item>

			<item>
				<widget class="QWidget" name="contentArea" native="true">
					<layout class="QVBoxLayout" name="verticalLayout_2">
						<item>
							<spacer name="verticalSpacer_top">
								<property name="orientation">
									<enum>Qt::Vertical</enum>
								</property>
								<property name="sizeHint" stdset="0">
									<size>
										<width>20</width>
										<height>40</height>
									</size>
								</property>
							</spacer>
						</item>

						<item>
							<widget class="QLabel" name="pageTitle">
								<property name="text">
									<string>Modify Account</string>
								</property>
								<property name="alignment">
									<set>Qt::AlignCenter</set>
								</property>
							</widget>
						</item>

						<item>
							<layout class="QHBoxLayout" name="horizontalLayout_3">
								<item>
									<spacer name="horizontalSpacer_left">
										<property name="orientation">
											<enum>Qt::Horizontal</enum>
										</property>
										<property name="sizeHint" stdset="0">
											<size>
												<width>40</width>
												<height>20</height>
											</size>
										</property>
									</spacer>
								</item>

								<item>
									<widget class="QFrame" name="modifyCard">
										<property name="minimumSize">
											<size>
												<width>400</width>
												<height>480</height>
											</size>
										</property>
										<property name="maximumSize">
											<size>
												<width>400</width>
												<height>550</height>
											</size>
										</property>
										<layout class="QVBoxLayout" name="verticalLayout_3">
											<property name="spacing">
												<number>15</number>
											</property>
											<property name="leftMargin">
												<number>30</number>
											</property>
											<property name="topMargin">
												<number>30</number>
											</property>
											<property name="rightMargin">
												<number>30</number>
											</property>
											<property name="bottomMargin">
												<number>30</number>
											</property>

											<item>
												<widget class="QLabel" name="labelAccNum">
													<property name="text">
														<string>Account Number</string>
													</property>
												</widget>
											</item>
											<item>
												<widget class="QLineEdit" name="accNumEdit">
													<property name="text">
														<string>123456789</string>
													</property>
													<property name="readOnly">
														<bool>true</bool>
													</property>
												</widget>
											</item>

											<item>
												<widget class="QLabel" name="labelName">
													<property name="text">
														<string>Full Name</string>
													</property>
												</widget>
											</item>
											<item>
												<layout class="QHBoxLayout" name="horizontalLayout_Name">
													<property name="spacing">
														<number>10</number>
													</property>
													<item>
														<widget class="QLineEdit" name="nameEdit">
															<property name="text">
																<string>John Doe</string>
															</property>
														</widget>
													</item>
													<item>
														<widget class="QPushButton" name="updateNameBtn">
															<property name="text">
																<string>Update</string>
															</property>
															<property name="class" stdset="0">
																<string>updateBtn</string>
															</property>
														</widget>
													</item>
												</layout>
											</item>

											<item>
												<widget class="QLabel" name="labelPass">
													<property name="text">
														<string>Password</string>
													</property>
												</widget>
											</item>
											<item>
												<layout class="QHBoxLayout" name="horizontalLayout_Pass">
													<property name="spacing">
														<number>10</number>
													</property>
													<item>
														<widget class="QLineEdit" name="passEdit">
															<property name="text">
																<string>Password123</string>
															</property>
															<property name="echoMode">
																<enum>QLineEdit::Password</enum>
															</property>
														</widget>
													</item>
													<item>
														<widget class="QPushButton" name="updatePassBtn">
															<property name="text">
																<string>Update</string>
															</property>
															<property name="class" stdset="0">
																<string>updateBtn</string>
															</property>
														</widget>
													</item>
												</layout>
											</item>

											<item>
												<widget class="QLabel" name="labelAddress">
													<property name="text">
														<string>Address</string>
													</property>
												</widget>
											</item>
											<item>
												<layout class="QHBoxLayout" name="horizontalLayout_Addr">
													<property name="spacing">
														<number>10</number>
													</property>
													<item>
														<widget class="QLineEdit" name="addressEdit">
															<property name="text">
																<string>123 Main St, Anytown, USA</string>
															</property>
														</widget>
													</item>
													<item>
														<widget class="QPushButton" name="updateAddrBtn">
															<property name="text">
																<string>Update</string>
															</property>
															<property name="class" stdset="0">
																<string>updateBtn</string>
															</property>
														</widget>
													</item>
												</layout>
											</item>

											<item>
												<spacer name="verticalSpacer_Card">
													<property name="orientation">
														<enum>Qt::Vertical</enum>
													</property>
													<property name="sizeHint" stdset="0">
														<size>
															<width>20</width>
															<height>20</height>
														</size>
													</property>
												</spacer>
											</item>

											<item>
												<widget class="QPushButton" name="saveBtn">
													<property name="text">
														<string>Save Changes</string>
													</property>
													<property name="cursor">
														<cursorShape>PointingHandCursor</cursorShape>
													</property>
												</widget>
											</item>
										</layout>
									</widget>
								</item>

								<item>
									<spacer name="horizontalSpacer_right">
										<property name="orientation">
											<enum>Qt::Horizontal</enum>
										</property>
										<property name="sizeHint" stdset="0">
											<size>
												<width>40</width>
												<height>20</height>
											</size>
										</property>
									</spacer>
								</item>
							</layout>
						</item>

						<item>
							<spacer name="verticalSpacer_bottom">
								<property name="orientation">
									<enum>Qt::Vertical</enum>
								</property>
								<property name="sizeHint" stdset="0">
									<size>
										<width>20</width>
										<height>40</height>
									</size>
								</property>
							</spacer>
						</item>
					</layout>
				</widget>
			</item>

			<item>
				<widget class="QFrame" name="footerFrame">
					<property name="minimumSize">
						<size>
							<width>0</width>
							<height>50</height>
						</size>
					</property>
					<layout class="QHBoxLayout" name="horizontalLayout">
						<property name="leftMargin">
							<number>40</number>
						</property>
						<property name="rightMargin">
							<number>40</number>
						</property>
						<item>
							<widget class="QLabel" name="badgesLabel">
								<property name="text">
									<string>üõ°Ô∏è Security Guaranteed</string>
								</property>
								<property name="styleSheet">
									<string>color: white;</string>
								</property>
							</widget>
						</item>
						<item>
							<spacer name="horizontalSpacer_4">
								<property name="orientation">
									<enum>Qt::Horizontal</enum>
								</property>
								<property name="sizeHint" stdset="0">
									<size>
										<width>40</width>
										<height>20</height>
									</size>
								</property>
							</spacer>
						</item>
						<item>
							<widget class="QLabel" name="footerLinks">
								<property name="text">
									<string>Privacy Policy    Contact Us</string>
								</property>
								<property name="alignment">
									<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
								</property>
							</widget>
						</item>
					</layout>
				</widget>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections/>
</ui>
